tasks/build/package_scripts/post_remove.sh

```diff
@@ -9,10 +9,34 @@ user_remove() {
   userdel "$1"
 }
 
+REMOVE_USER=false
+
 case $1 in
-    purge|0)
-        if user_check "<%= user %>"  ; then
-          user_remove "<%= user %>"
-        fi
-        ;;
+
+  # Debian
+  purge)
+    REMOVE_USER=true
+  ;;
+
+  remove|failed-upgrade|abort-install|abort-upgrade|disappear|upgrade|disappear)
+  ;;
```

Comment:
There's a catch all at the bottom that echos an error if invalid arguments are used, these are valid arguments we just aren't doing anything with them